### PR TITLE
WIP: Visual design overhaul and v1 release

### PR DESCRIPTION
**Pull request by designmatty**
### General updates

- [x] Remove chart package
- [x] Update base theme
- [x] Removed pill component

### Primitives updates
- [x] Box
  - [x] styles/theme
  - [x] documentation
  - [x] reviewed/approved

### Typography updates
* [ ] A
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

- [x] Header
  - [x] styles/theme
  - [x] documentation
  - [x] reviewed/approved

* [x] Text
  - [x] styles/theme
  - [x] documentation
  - [x] reviewed/approved

### Components updates
- [ ] Badge
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

~* [ ] Pill~
  ~- [ ] styles/theme~
  ~- [ ] documentation~
  ~- [ ] reviewed/approved~

- [ ] Swatch
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

* [ ] Button
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

- [ ] Callout
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

* [x] Card
  - [x] styles/theme
  - [x] documentation
  - [x] reviewed/approved

- [x] Divider
  - [x] styles/theme
  - [x] documentation
  - [x] reviewed/approved

* [ ] Grouper
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

- [ ] Icon
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

* [ ] Image
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

- [x] ProgressBar
  - [x] styles/theme
  - [x] documentation
  - [x] reviewed/approved

* [ ] Table
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

### Form updates
- [ ] Checkbox
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

* [ ] ColorInput
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

- [ ] Field
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

* [ ] FileInput
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

- [ ] Label
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

* [ ] Radio
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

- [ ] Select
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

* [ ] TextInput
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

- [ ] TextArea
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

* [ ] ToggleField
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

### Navigation updates
- [ ] Brand
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

* [ ] Breadcrumbs
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

- [ ] NavBar
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

### Overlays updates
* [ ] Dialog
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

- [ ] Tooltip
  - [ ] styles/theme
  - [ ] documentation
  - [ ] reviewed/approved

### Checklist

- [ ] Relevant documentation pages have been created or updated
- [ ] Description of PR is in an appropriate section of the changelog and grouped with similar changes if possible

### CODE REVIEW

File: packages/core/components/header/index.mdx [hunk 1]
```diff
@@ -1,30 +1,80 @@
...
 import Header from './'
 
 # Header
 
+## Import
+`import { Header } from '@blasterjs/core'`
+
+## Basic Usage
+
+Transposes [`Text`](/typography/text) and renders an `<h1>` by default.
```
Comment (alkamin) [inline packages/core/components/header/index.mdx]: 'Composes' instead of 'Transposes'

File: packages/core/components/divider/index.mdx [hunk 2]
```diff
@@ -10,31 +10,81 @@ import { Direction } from "../../common/direction";
...
+
+## Basic Usage
+
+By default, the `Divider` component transposes [`Box`](/primitives/) with a default prop `tag=hr` which will render a horionzal rule.
+
+<Playground>
+  <Divider />
+</Playground>
+
+## Veritcal Divider
```
Comment (alkamin) [inline packages/core/components/divider/index.mdx]:
```suggestion
## Vertical Divider
```

File: packages/core/components/progressBar/index.mdx [hunk 3]
```diff
@@ -10,12 +10,42 @@ import ProgressBar from "./";
...
+## Basic Usage
+
 Displays a fractional amount of progress towards some end.
 
 <Playground>
   <ProgressBar value={25} max={100} />
 </Playground>
 
+## Striped and animated
+Use the `striped` and `animated` prop to change the default appearance and show motion.
```
Comment (alkamin) [inline packages/core/components/progressBar/index.mdx:26]: May be worth mentioning that `animated` has no visible effect unless `striped` is present.

File: packages/core/components/progressBar/index.mdx [hunk 4]
```diff
@@ -28,3 +58,30 @@ Displays a fractional amount of progress towards some end.
...
+    height
+  },
+  styles: {
+    backBar: css``,
+    topBar: css``
+  }
+}
+```
+
+- [Default theme](https://github.com/raster-foundry/blasterjs/tree/master/packages/core/theme/components/progressBar)
```
Comment (alkamin) [inline packages/core/components/progressBar/index.mdx]:
```suggestion
- [Default theme](https://github.com/raster-foundry/blasterjs/tree/master/packages/core/theme/components/progressBar/index.js)
```

File: packages/core/components/progressBar/index.mdx [hunk 5]
```diff
@@ -28,3 +58,30 @@ Displays a fractional amount of progress towards some end.
...
+- [Learn more about themeing](#)
```
Comment (alkamin) [inline packages/core/components/progressBar/index.mdx:93]: Probably could be removed until we have theming docs

File: packages/core/components/progressBar/index.mdx (hunk 3 above)
Comment (designmatty) [reply packages/core/components/progressBar/index.mdx:26]: I added a callout

File: packages/core/components/progressBar/index.mdx (hunk 5 above)
Comment (designmatty) [reply packages/core/components/progressBar/index.mdx:93]: This is throughout the docs. I will remove this line in all docs in another pr.

**Review by alkamin — APPROVED** [review]

**Pull request merged**